clients/githubrepo/graphql.go

```diff
@@ -152,6 +173,24 @@ func pullRequestFrom(data *graphqlData) []clients.PullRequest {
 	return ret
 }
 
+func releasesFrom(data *graphqlData) []clients.Release {
+	releases := make([]clients.Release, 0)
```

Comment:
nit: do we need the ", 0" (or the make() at all for that matter)? https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices also says to prefer `var releases []clients.Release`

Reply:
Done.